src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx

```diff
@@ -31,6 +35,16 @@ const EditQuizContainer = ({ questions, setQuestions }: QuizContentProps) => {
   const [title, setTitle] = useState<string>('')
   const [description, setDescription] = useState<string>('')
   const [isCreationOpen, setIsCreationOpen] = useState<boolean>(false)
+  const [searchParams, setSearchParams] = useSearchParams()
```

Comment:
you don`t need to make any manipulation with url, it should be state like with title and desription

Reply:
done